sc_server_auth/server/hashing.py

```diff
@@ -0,0 +1,14 @@
+import hashlib
+
+_HASH_NAME = "sha256"
+_SALT = b"\xf2\x0fr{\x98\x12\x9f\x08'\xfd_\x1f-sF\x84"
```

Comment:
Is it really a good idea store salt here?

Reply:
It can be in toml settings file

Reply:
Done